utm.yaml

```diff
@@ -3457,6 +3491,20 @@ paths:
         $ref: '#/components/schemas/EntityID'
       in: path
       required: true
+    - name: version
+      description: >-
+        This optional field not part of the original F3548 standard API allows a
+        remote USS to request a specific version of the operational intent
+        details.  USSs are not required to implement support for this field.  If
+        implemented, the USS must make available the published and the pending
+        versions of the operational intent.  It is not required for older
+        versions to be available.
+      in: query
+      required: false
+      schema:
+        $ref: '#/components/schemas/EntityVersion'
+    description: >-
+      The number of items to skip before starting to collect the result set
```

Comment:
I think this probably needs to be cleaned out

Reply:
Indeed, not sure how that slipped in here.